src/main/java/uk/gov/hmcts/divorce/caseworker/service/CcdSearchService.java

```diff
@@ -0,0 +1,81 @@
+package uk.gov.hmcts.divorce.caseworker.service;
+
+import feign.FeignException;
+import lombok.extern.slf4j.Slf4j;
+import org.elasticsearch.search.builder.SearchSourceBuilder;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.divorce.common.model.State;
+import uk.gov.hmcts.divorce.idam.IdamService;
+import uk.gov.hmcts.reform.authorisation.generators.AuthTokenGenerator;
+import uk.gov.hmcts.reform.ccd.client.CoreCaseDataApi;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.ccd.client.model.SearchResult;
+import uk.gov.hmcts.reform.idam.client.models.User;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.elasticsearch.index.query.QueryBuilders.boolQuery;
+import static org.elasticsearch.index.query.QueryBuilders.matchQuery;
+import static org.elasticsearch.search.sort.SortOrder.ASC;
+import static uk.gov.hmcts.divorce.ccd.NoFaultDivorce.CASE_TYPE;
+
+@Service
+@Slf4j
+public class CcdSearchService {
+
+    public static final int PAGE_SIZE = 100;
```

Comment:
Perhaps this should be configurable. I'm just thinking that if we ever went over 100 (which seems likely) it would be difficult to track down what is happening.